naga/src/valid/mod.rs

```diff
@@ -237,7 +237,7 @@ bitflags::bitflags! {
     }
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Default)]
```

Comment:
I don't think we want this in general, as you should need to validate a module to get a module info.